src/components/CartItem/CartItem.tsx

```diff
@@ -5,6 +5,7 @@ import DeleteIcon from '../Icon/DeleteIcon';
 import Button from '../Button/Button';
 import { useState } from 'react';
 import unAvailableImage from '../../assets/images/no-image.jpg';
+import Loading from '../Loading';
```

Comment:
```
import Button from '../Button/Button'
import Loading from '../Loading'
```
Please consistent import, you can fix in another PR

Reply:
Fixed